.eslintrc.json

```diff
@@ -17,10 +19,20 @@
 		"prettier/@typescript-eslint"
 	],
 	"rules": {
-		"no-console": "off",
+		"github/no-then": "off", // TODO enable.
+		"import/no-namespace": "off", // TODO enable.
+		"no-console": "off", // TODO enable.
```

Comment:
I actually don't need to disable `import/no-namespace`, I think I removed the error-causing code. For `github/no-then` we need to stop using `.then()`. For `no-console` we need to stop logging using `console`.

Reply:
A rule similar to `github/no-then` is already included in `eslint-plugin-promises` so we can probably remove it here. Also it doesn't always make sense to avoid "then", especially in top-level code where you can't use `await`.

For no-console: good point. When you make the PR to add directory structure, we can go ahead and remove this override, cause it's only needed for our program (non-module) code.

Reply:
Since I stopped extending GitHub, I don't even need to override it anymore.